README.md

```diff
@@ -312,14 +312,28 @@ you must also configure a custom domain, and set it as the `baseUrl` prop in the
 
 You can find an example react app in the [examples folder](./examples).
 
+### Widgets
```

Comment:
This section should be in the bottom of the examples section 

Widgets are secondary to other sdk api (auth provides , hooks, flows) and also require previous steps so that it will work